crates/rpc/rpc-builder/src/lib.rs

```diff
@@ -1298,13 +1285,17 @@ impl RpcServerConfig {
         self.jwt_secret.map(|secret| AuthLayer::new(JwtAuthValidator::new(secret)))
     }
 
-    /// Builds the ws and http server(s).
+    /// Builds and starts the ws and http server(s).
     ///
     /// If both are on the same port, they are combined into one server.
-    async fn build_ws_http(
+    pub async fn start(
         &mut self,
```

Comment:
I think this should now just consume the type
```suggestion
        self,
```